inv/inventory_test.go

```diff
@@ -75,6 +76,13 @@ func TestInventoryListDevices(t *testing.T) {
 			outError:        errors.New("failed to fetch devices: db connection failed"),
 			outDevices:      nil,
 		},
+		"get devices from group": {
```

Comment:
I am unsure as to the validity of this test. It passes, but how is it actually working, with the whole mocking functionality and all?